data/meetings/d10-friends-of-bill-w.yaml

```diff
@@ -0,0 +1,10 @@
+{'address': '1537 Baltimore Ave, Hot Springs, SD, 57747',
+ 'area': 63,
+ 'district': 10,
+ 'latitude': 43.427498255981014,
+ 'longitude': -103.47945082703903,
+ 'name': 'Friends of Bill W',
+ 'note': 'Enter door to left of Main Entrance downstairs(lift provided)',
+ 'place': 'Bethesda Lutheran Church',
+ 'time': {'Saturday': ['10:00'], 'Thursday': ['18:00'], 'Tuesday': ['18:00']},
```

Comment:
Please follow the same format/structure found in the other yaml files.

Reply:
Okay I did copy/paste this from the parser? I will make changes.